.github/workflows/pr-highlight-changes.yml

```diff
@@ -26,7 +27,7 @@ jobs:
           GIT_CLONE_PROTECTION_ACTIVE: false
         run: |
           HEAD_REF=$(git rev-parse HEAD)
-          exclude="plugins/woocommerce/tests plugins/woocommerce-admin/tests plugins/woocommerce-blocks/tests"
+          exclude="plugins/woocommerce/tests plugins/woocommerce/templates/templates/ plugins/woocommerce-admin/tests plugins/woocommerce-blocks/tests"
```

Comment:
Nit: for the sake of consistency, let's drop the trailing slash in `plugins/woocommerce/templates/templates/`